styles.css

```diff
@@ -0,0 +1,42 @@
+body {
+  background-color: #E96D65;
+  margin: 50px;
+  font-family: 'Lato', sans-serif;
+  color: #fff;
+  font-size: 2.5em;
+}
+.input {
+  position: relative;
+}
+input {
+  background-color: #E96D65;
+  border: none;
+  border-bottom: 1px dotted #b14943;
+  color: #b14943;
+}
+
+input[type=number]:focus {
+  background-color: #76C3BD;
+  border: none;
+  outline-style: none;
+  color: #fff;
+  transform: scale(3);
+  transition: 0.5s;
+}
+.button {
```

Comment:
Remember to add `position: relative` so that you can use `position: absolute` in the pseudo-element :+1:

Reply:
I put the relative as the input as that seem to make better sense to me.
I have now moved it to .button as you have suggested here.